### DISCUSSION TITLE
POST /api/v6/domains (400, ERROR: The registrant_type must be specified)

### DISCUSSION THREAD

**Discussion by varna** [vercel/community#2537] [Help] [answered]
Hi, I tried to Transfer In a domain with `.lt` ending from local provider (iv.lt) to Vercel via web UI: https://vercel.com/dashboard/domains
<img width="541" alt="image" src="https://github.com/vercel/community/assets/6717694/c12ce628-580d-4471-bb42-2ddafeba7853">
But it fails with `POST /api/v6/domains (400, ERROR: The registrant_type must be specified)` error:
<img width="556" alt="image" src="https://github.com/vercel/community/assets/6717694/e5ddf992-deb5-43b2-b143-330c1b225b87">

**Comment by smaeda-ks** [comment, marked as answer]
Hi @varna, unfortunately,`.lt` is not supported for transferring in. But you can still use Vercel name servers if you want:
https://vercel.com/docs/concepts/projects/domains#nameservers

As mentioned above, you can change your authoritative name servers to ours.
